bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -21,307 +22,349 @@
 {% block page_title_suffix %}{% endblock %}
 
 {% block body_id %}notes{% endblock %}
-{% block body_class %}fx-notes{% endblock %}
+{% block body_class %}mzp-t-firefox {% if version.split('.')[0] | int < 57 %}t-pre-quantum {% endif %}{% endblock %}
```

Comment:
@pmac does this look okay to you? The colour coding in my editor goes wonky at this point but it runs.

Reply:
it does look okay, but there is a better value for you available for this:

```jinja
{% if release.major_version_int < 57 %}t-pre-quantum {% endif %}
```

The `release` variable in the context is an instance of the `ProductRelease` model class. The properties available are on the [ProductRelease](https://github.com/mozilla/bedrock/blob/6858eb808894d6d5ee753689ca1bd46d10f89bb2/bedrock/releasenotes/models.py#L177-L187) model.

Reply:
Also be sure this isn't being used for Fx for iOS since those versions are all below 57.